salt/iiif/config/etc-loris2-uwsgi.ini

```diff
@@ -2,51 +2,26 @@
 chdir=/opt/loris/
```

Comment:
This will probably change a lot, but it's right to mount it as configuration on the container, as it contains tunable values such as `processes`

Reply:
the `uwsgi.ini` in all of our projects could do with *a lot* of optimisation to be honest. I hope we do find time to tweak the settings beyond simply just working.

>  tunable values such as processes

This probably shouldn't be hardcoded but derived from the number of cores available... again, something that would be fun to tune later on